modules/olmv1-catalog-exclusion-by-labels-or-expressions.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+// * extensions/catalogs/olmv1-catalog-content-resolution.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="olmv1-catalog-exclusion-by-labels-or-expressions_{context}"]
+= Catalog exclusion by labels or expressions
+
+You can exclude catalogs by using match expressions on metadata with the `NotIn` or `DoesNotExist` operators.
+
+The following cluster catalog CR adds the `example.com/testing` label to the `unwanted-catalog` cluster catalog:
+
+.Example cluster catalog CR
+[source,yaml]
+----
+apiVersion: olm.operatorframework.io/v1
+kind: ClusterCatalog
+metadata:
+  name: unwanted-catalog
+  labels:
+    example.com/testing: "true"
+spec:
+  source:
+    type: Image
+    image:
+      ref: quay.io/example/content-management-a:latest
+----
+
+The following cluster extension CR excludes selection from the `unwanted-catalog` catalog:
```

Comment:
Do these two examples essenatially do the same thing: not select from the `unwanted-catalog`? If so, why would someone choose one over the other?

Reply:
Great feedback. I added a second catalog with the same label. Now, the first example excludes `unwanted-catalog-1`  and the second example excludes `unwanted-catalog-1` and `unwanted-catalog-2`.